source/boundary_temperature/dynamic_core.cc

```diff
@@ -128,24 +128,24 @@ namespace aspect
                              "The max iterations for nonliner core energy solver.");
           prm.declare_entry ("Core heat capacity", "840",
                              Patterns::Double (0),
-                             "Heat capacity of the core. Units: $J/kg/K$");
+                             "Heat capacity of the core. Units: $J/kg/K$.");
           prm.declare_entry ("K0", "4.111e11",
                              Patterns::Double (0),
                              "Core compressibility at zero pressure. "
                              "Referring to Nimmo et al. (2004) for more details.");
           prm.declare_entry ("Rho0", "7.019e3",
                              Patterns::Double (0),
-                             "Core density at zero pressure. Units: $kg/m^3$. "
+                             "Core density at zero pressure. Units: $kg/m^3$."
```

Comment:
Can you add that space again?

Reply:
Oh, I totally missed the following sentence -- we always have the units at the end.

Fixed! (I also fixed language slightly.)